docs/physics/setup/plasma/index.rst

```diff
@@ -37,7 +37,10 @@ Every property has a `calculate` function that returns the values of its outputs
 
 The Plasma Graph
 ----------------
-If the necessary Python modules (`PyGraphviz <https://pygraphviz.github.io/>`_ and `dot2tex <https://dot2tex.readthedocs.io/en/latest/>`_) are available, TARDIS will automatically output a .tex file at the beginning of each run that can be compiled to produce a PDF image of the plasma module graph. The nodes on this graph are the names of plasma properties, e.g. `Levels`, `TauSobolev`, `IonNumberDensity`, along with a list of outputs from those properties and equations showing how they are calculated. These nodes are connected by arrows linking nodes with the sources of their inputs, and labelled with the name of the input/output linking the two properties, e.g. `levels`, :math:`\tau_{\textrm{sobolev}}`, :math:`n_{e}`.
+If the necessary Python modules (`PyGraphviz <https://pygraphviz.github.io/>`_ and `dot2tex <https://dot2tex.readthedocs.io/en/latest/>`_) are available, TARDIS will automatically output a .dot and a .tex file at the beginning of each run that can be compiled to produce a PDF image of the plasma module graph. The nodes on this graph are the names of plasma properties, e.g. `Levels`, `TauSobolev`, `IonNumberDensity`, along with a list of outputs from those properties. These nodes are connected by edges linking them with the sources of their inputs. The .tex file contains the name of the input/output linking the properties (e.g. `levels`, :math:`\tau_{\textrm{sobolev}}`, :math:`n_{e}`), as well as the equations used to calculate them, written in LaTeX. Tutorial coming soon!
```

Comment:
are the packages part of the conda environment?

Reply:
It looks as though PyGraphviz is within the conda-forge channel, however dot2tex is in the pypi channel and not conda-forge

Reply:
have a look at `tardis_env.yml` and see if they are in there.

Reply:
In there, all the packages (including PyGraphviz and dot2tex) are listed under the conda-forge channel, however dot2tex looks to be installed via pip as it is the only member within the pip list, but this too is listed as a dependency of the conda-forge channel. I presume this means that, although they are installed differently, they both fall under the conda environment?